minkindr/include/kindr/minimal/implementation/angle-axis-inl.h

```diff
@@ -62,6 +62,15 @@ AngleAxisTemplate<Scalar>::AngleAxisTemplate(const RotationMatrix& matrix) :
   // \todo furgalep Check that the matrix was good...
 }
 
+template<typename Scalar>
+AngleAxisTemplate<Scalar>::AngleAxisTemplate(const Vector3& angleAxis) {
+    Scalar norm = angleAxis.norm();
+    CHECK_LE(1e-8, norm);
```

Comment:
will turn it around. I would play safe and block the construction as the condition must be handled upstream anyways?